pkg/arvo/lib/pull-hook.hoon

```diff
@@ -1,17 +1,49 @@
+::  lib/pull-hook/hoon: helper for creating a push hook
+::  
+::   lib/pull-hook is a helper for automatically pulling data from a
+::   corresponding push-hook to a store. 
+::   
+::   ## Interfacing notes:
+::
+::   lib/push-hook provides the inner door with its own separate state.
+::   The inner door may interact with the library by producing cards or
+::   scrying. Do not pass any cards on a wire beginning with /helper as
```

Comment:
>or scrying

I don't see any special scry handling in this. If there was, you'd want to mention the prefix path used for these.

But also, this seems potentially problematic. At the very least you want to pass it onto the underlying agent, right?

https://github.com/urbit/urbit/blob/d91de5f2769060e81bd46c696f373064a9bf9033/pkg/arvo/lib/pull-hook.hoon#L246

Reply:
(Also `/helper` is a bit unhelpful as far as descriptive prefixes go. It's the generic name, but you might want to have multiple helper libraries that do different things, etc. etc.)

Reply:
Oh the plan was to have `/helper` be generally a reserved prefix for helper libraries. i.e. pull-hook uses /helper/pull-hook and push-hook uses /helper/push-hook as their wire prefixes.